linode/apl-core
# fix: Apl 307 apl-charts repo creation

This PR adds an extra check in the apl installation job which checks the presence of the apl-charts repo in gitea. It uses the `git ls-remote` command to do so and in case the repo is there it will skip cloning it again which was previously causing issues when the job did not finish in the first attempt.

Validation steps:
1. Deploy APL from this branch 
2. What for the installation job to start and monitor its logs
3. When you see this line in the logs: `otomi:cmd:commit:gitea-apl-charts:info Cloning apl-charts in Gitea`, then kill the pod so a new one resumes the installation
4. The second pod should finish the installation without errors
